packages/cfd/src/Containers/dmt5-trade-modal.tsx

```diff
@@ -49,6 +50,8 @@ const DMT5TradeModal = observer(
             account_status: { authentication },
         } = client;
 
+        const [url, setURL] = React.useState<string>('');
```

Comment:
```suggestion
        const [url, setURL] = React.useState('');
```
This is a primitive type.. u can just let TS infer this.